diff -Naur gamescope-3.16.3.old/src/meson.build gamescope-3.16.3/src/meson.build
--- gamescope-3.16.3.old/src/meson.build	2025-04-09 21:52:06.938808530 +0200
+++ gamescope-3.16.3/src/meson.build	2025-04-09 22:01:44.361532915 +0200
@@ -125,6 +125,7 @@
 
 luajit_dep = dependency( 'luajit' )
 libinput_dep = dependency('libinput', required: true)
+libudev_dep = dependency('libudev', required: true)
 
 gamescope_cpp_args = []
 if drm_dep.found()
@@ -197,7 +198,7 @@
       xkbcommon, thread_dep, sdl2_dep, wlroots_dep,
       vulkan_dep, liftoff_dep, dep_xtst, dep_xmu, cap_dep, epoll_dep, pipewire_dep, librt_dep,
       stb_dep, displayinfo_dep, openvr_dep, dep_xcursor, avif_dep, dep_xi,
-      libdecor_dep, eis_dep, luajit_dep, libinput_dep, pixman_dep
+      libdecor_dep, eis_dep, luajit_dep, libinput_dep, pixman_dep, libudev_dep
     ],
     install: true,
     cpp_args: gamescope_cpp_args,
